lib/apiservers/engine/backends/container.go

```diff
@@ -906,6 +908,16 @@ func (c *Container) ContainerStop(name string, seconds *int) error {
 		return NotFoundError(name)
 	}
 
+	if seconds == nil {
+		var timeout int
+		if vc.Config.StopTimeout != nil {
+			timeout = *vc.Config.StopTimeout
+		} else {
+			timeout = DefaultStopTimeout
+		}
+		seconds = &timeout
+	}
+
```

Comment:
:) I considered this.  Being a C guy, I figured one less assignment was more efficient but fine.